src/services/vcmap/context.ts

```diff
@@ -21,32 +22,22 @@ const layers = [
     url: 'https://demo.virtualcitymap.de/rennes/datasource-data/f661c55d-d40b-44fb-889f-88176163cba2',
     activeOnStartup: true,
   }),
-  new WMSLayer({
+  new GeoJSONLayer({
```

Comment:
Be careful, TRAM/Bus, Parking on POI are not meant to replace bus/metro/bike layers.
They are vector layer which can't be removed manually from the map.